### PR TITLE
add menu popup

### PR DESCRIPTION
**Pull request by alizaamir123**
##In this PR:
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -50,7 +50,7 @@ body {
 .menue-popup {
```
Comment (Kossi-stack) [inline style.css:50]: - [ ] Kindly increase the size of the hamburger
- Your Design
Screenshot 2023-05-23 at 13 18 48
- Figma Design
Screenshot 2023-05-23 at 13 24 50

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @alizaamir123,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly make the Live Demo a link in the Readme file.
 
 NB:  Next time kindly provide the readme file for review purposes.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -13,62 +13,12 @@
...
-            <li class="html technologies1">Html</li>
-            <li class="Css technologies1">Css</li>
-            <li class="javascript technologies1">Javascript</li>
-            <li class="pophidden hidden technologies1">Github</li>
-            <li class="pophidden  hidden technologies1">Ruby</li>
-            <li class="pophidden hidden technologies1">Bootstrap</li>
-        </ul>
-        <div class="underline"></div>
-        <div class="pop-button">
-          <button class="see-project pop-project"><a href="#">
```
Comment (stanleySimeon) [inline index.html:51]: Kindly I suggest you add the button type to follow HTML's good semantics.

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
Hi @alizaamir123, @wwwdrt, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] As mentioned by the previous reviewer, kindly provide the readme file for review purposes.
- [OPTIONAL] As mentioned by the previous reviewer,I highly recommend you to increase the size of the hamburger.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
Hi @alizaamir123, @wwwdrt, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] As mentioned by the previous reviewer, kindly provide the readme file for review purposes.
- [OPTIONAL] As mentioned by the previous reviewer,I highly recommend you to increase the size of the hamburger.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @alizaamir123  

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights 
- Kudus to heed the required changes as stated previously  👍 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**